src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,44 +1,68 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo,
+  selectMethod: (value: null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  selectMethod,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const getUserFromServer = async () => {
+    setUser(await getUser(selectedTodo.userId));
```

Comment:
why did you remove try/catch statement? it was just suggested to add some value to state. And then you could use this value for showing some error(title) on your page.